pkg/trace/agent/obfuscate_test.go

```diff
@@ -50,6 +52,84 @@ func TestMetaHook(t *testing.T) {
 	}
 }
 
+func TestAppSecMetaStructHook(t *testing.T) {
+	cco := newCreditCardsObfuscator(config.CreditCardsConfig{Enabled: true})
+	defer cco.Stop()
+
+	t.Run("no creditcards", func(t *testing.T) {
+
+		appsec := pb.AppSecStruct{Triggers: []*pb.AppSecTrigger{{
```

Comment:
I find it quite confusing to have a variable name be the same as a package name. Perhaps you get a better idea. I don't have one personally 🙂